tangelo/linq/simulator.py

```diff
@@ -86,7 +87,7 @@ def __init__(self, target=default_simulator, n_shots=None, noise_model=None):
         if not self.n_shots and (not self.statevector_available or self._noise_model):
             raise ValueError("A number of shots needs to be specified.")
 
-    def simulate(self, source_circuit, return_statevector=False, initial_statevector=None):
+    def simulate(self, source_circuit, return_statevector=False, initial_statevector=None, qubits_to_use=None, opt_level=1, meas_mitt=False):
```

Comment:
I don't think these options belong here at first glance, I think they should be attributes of a backend object.
We want ideally a clear separation between the object that define what the computation is, and whatever is backend related.